src/components/Dashboard/index.js

```diff
@@ -26,6 +26,10 @@ class Dashboard extends React.Component {
     this.setState(utils.checkDevice());
   }
 
+  componentDidMount() {
+    this.props.embed ? document.getElementsByClassName('dashboard-header')[0].style.justifyContent= 'flex-end' :'';
```

Comment:
I think is better to add a class to manage those style changes instead of adding those styles directly. 
And you don't even need that class because you can manage everything with the main "is-embed" class that we add to .l-app.